### DISCUSSION TITLE
Symfony 6.1 my login form does not connect me no error message

### DISCUSSION THREAD

**Discussion by kawick-dev** [symfony/symfony#47314] [Q&A] [answered]
Hello,
I have a problem with my login form, I can't connect, I have no error message, it just doesn't connect me.

I followed the symfony doc to create the connection
https://symfony.com/doc/current/security.html

The login form works in locally but does not work on a server hosted by O2Switch
After some test I noticed that it passes well in SymfonyComponentSecurityHttpAuthentication::DefaultAuthenticationSuccessHandler

MY FILES : 
**Login Controller**
`<?php

namespace App\Controller;
use Symfony\Component\Security\Http\Authentication\AuthenticationUtils;
use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
use Symfony\Component\HttpFoundation\Response;
use Symfony\Component\Routing\Annotation\Route;

class LoginController extends AbstractController
{
    #[Route('/login', name: 'app_login')]
    public function index(AuthenticationUtils $authenticationUtils): Response
    {
        $error = $authenticationUtils->getLastAuthenticationError();
        $lastUsername = $authenticationUtils->getLastUsername();
        return $this->render('login/index.html.twig', [
            'last_username' => $lastUsername,
            'error'         => $error,
        ]);
    }
}
`

**security.yaml**
`security:
    # https://symfony.com/doc/current/security.html#registering-the-user-hashing-passwords
    password_hashers:
        Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface: 'auto'
    # https://symfony.com/doc/current/security.html#loading-the-user-the-user-provider
    providers:
        # used to reload user from session & other features (e.g. switch_user)
        app_user_provider:
            entity:
                class: App\Entity\User
                property: email
    firewalls:
        dev:
            pattern: ^/(_(profiler|wdt)|css|images|js)/
            security: false
        main:
            lazy: true
            provider: app_user_provider
            form_login:
                # "app_login" is the name of the route created previously
                login_path: app_login
                check_path: app_login
                enable_csrf: true
            logout:
                path: app_logout
                target: app_home

            # activate different ways to authenticate
            # https://symfony.com/doc/current/security.html#the-firewall

            # https://symfony.com/doc/current/security/impersonating_user.html
            # switch_user: true

    # Easy way to control access for large sections of your site
    # Note: Only the *first* access control that matches will be used
    access_control:
        # - { path: ^/admin, roles: ROLE_ADMIN }
        # - { path: ^/profile, roles: ROLE_USER }

when@test:
    security:
        password_hashers:
            Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface:
                algorithm: auto
                cost: 4 # Lowest possible value for bcrypt
                time_cost: 3 # Lowest possible value for argon
                memory_cost: 10 # Lowest possible value for argon
`
Thank you in advance for your help

**Comment by OjicA** [comment, marked as answer]
Hey, I am a co-worker of @kawick-dev, and we just found the response. In fact, the user get logged but log out instantly when the page refresh. We get a message: "Cannot refresh token because user has changed." . 
If you happen to get the same issue, go to your User Entity and implemente the "EquatableInterface" with the method isEqualTo(), if worked for us.
